jenkins/automation/release-process.groovy

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env groovy
+pipeline {
+
+    agent {
+        node {
+            label 'docker-centos-7.9.2009-node'
+        }
+    }
+
+    parameters {
+        string(name: 'BASE_TAG', defaultValue: '', description: 'Image tag which need to be tagged with TARGET_TAG. It is used to tag component commits and create GitHub Release as well')
+        string(name: 'TARGET_TAG', defaultValue: '', description: 'New tag to be tagged on Image')
+        string(name: 'SERVICES_RELEASE', defaultValue: '', description: 'Services(cortx-k8s) release version on which image deployment is tested')
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch or GitHash for tagging/Changelog/GitHub Release scripts', trim: true)
+        string(name: 'CORTX_RE_REPO', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repository for tagging/Changelog/GitHub Release scripts', trim: true)
+        choice (
+            choices: ['DEVOPS', 'DEBUG'],
+            description: 'Email Notification Recipients ',
+            name: 'EMAIL_RECIPIENTS'
+        )
+    }
+
+    environment {
+        GITHUB_TOKEN = credentials('cortx-admin-token')
+        REGISTRY = "ghcr.io"
+        OWNER = "seagate"
+        LATEST_GH_SERVER_IMAGE_TAG = sh( script: """
+            curl -s -H \"Accept: application/vnd.github+json\" -H \"Authorization: token ${GITHUB_TOKEN}\" \"https://api.github.com/repos/${OWNER}/cortx/releases/latest\" | jq '.tag_name' | tr -d '\"'
```

Comment:
This call is fetching from the latest production release i.e. without `pre-release`. i.e. https://github.com/Seagate/cortx/releases/tag/2.0.0-790
```
[root@ssc-vm-g3-rhev4-1671 metrics-server]#  curl -s -H "Accept: application/vnd.github+json" -H "Authorization: token ${GITHUB_TOKEN}" "https://api.github.com/repos/seagate/cortx/releases/latest" | jq '.tag_name'
"2.0.0-790"
```
We will need to latest pre-prod release i.e. https://github.com/Seagate/cortx/releases/tag/2.0.0-895

Reply:
Fixed it